docs/getting-started/quickstart.mdx

```diff
@@ -39,10 +43,12 @@ import InstallTheCli from "@site/docs/_install-the-cli.mdx";
 
    <InstallTheCli />
 
-2. **Install [Docker](https://docs.docker.com/engine/install/)** (needed for local development)
```

Comment:
Removing this as `ddn doctor` performs this check and informs the user. We can also keep the docer pre-req here if it makes it explicit for the user

Reply:
Done